covfee/server/app.py

```diff
@@ -26,7 +26,9 @@
 from typing import Optional
 
 
-def create_app_and_socketio(mode="deploy", session_local=None):
+def create_app_and_socketio(
```

Comment:
Missing output typehints

Reply:
Added for mode in 7cd93b1

The return values are complex... this function seems to return the socketio module itself as it is using lazy import.  Thus the return type is unknown in the outer scope.